### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joelvbernier**
Fixes: #765

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
-# Develop
+# Installing
 
-Requires Python 3.8+
+## conda (release)
+To install the latest stable release
+
+```bash
+conda install -c hexrd -c anaconda -c conda-forge hexrdgui
```
Comment (psavery) [inline README.md:7]: We might want to clarify somewhere around here that python 3.8+ is required.

**Comment by psavery** [comment]
@joelvbernier I pushed a commit here that fixes the fabio error, and I also updated the instructions to add the `--no-build-isolation` flag which fixes an issue I encountered as part of #765.

With these changes, and with hexrd/hexrd#183, I am able to follow the instructions exactly and get a conda development
environment set up on macos.

File: README.md (hunk 1 above)
Comment (cjh1) [reply README.md:7]: Yes, lets not remove the fact that we need python 3.8, unless we want to support multiple versions ...

Comment (psavery) [reply README.md:9]: I just fixed that.

**Review by psavery — APPROVED** [review]
LGTM

File: setup.py [hunk 2]
```diff
@@ -16,8 +15,10 @@
 # dist info so setuptools can't find it, even though its there, which results in
 # pkg_resources.DistributionNotFound, even though the package is available. So we
 # only added it if we aren't building with conda.
+# fabio from the hexrd channel has a similar problem.
```
Comment (cjh1) [inline setup.py:18]: I have removed fabio from our channel, so I think this comment can me removed.

Comment (psavery) [reply setup.py:18]: I'll give it a try

Comment (psavery) [reply setup.py:18]: Yep, I just gave it a try, and this change is no longer needed. I'll revert this commit.

Comment (psavery) [reply setup.py]: @cjh1 That commit was removed, and now we are back to README only changes.

**Review by cjh1 — APPROVED** [review]

**Comment by joelvbernier** [comment]
Just to be clear, is Python 3.8 ok on Mac OS now?  Or do we still need our special python 3.7.6 for Qt?

**Comment by cjh1** [comment]
Our special python is 3.8. I haven't check recently but last time I looked we still needed it as the changes hadn't been back ported yet, I will take a look.

**Comment by psavery** [comment]
Yeah, I think once we start requiring python 3.9, we might not need the special python.

**Comment by cjh1** [comment]
OK, so I download 3.8.8 from conda-forge and it is indeed built with 10.15, so I think we are now safe to install from conda-forge, yay!

**Comment by cjh1** [comment]
But we need to update the readme to say the on the mac you need to install python from conda-forge, I will push a PR.

**Comment by joelvbernier** [comment]
Ah bollocks @cjh1 -- looks like the `conda-forge` python 3.8.8 is not working on Mac OS 11 (Big Sur); our old `HEXRD` python 3.8.4 is, however. Probabaly should fix the README (again) ASAP.

**Comment by cjh1** [comment]
@joelvbernier in what way does it not work? I can update the readme .

**Comment by cjh1** [comment]
@joelvbernier Is it just failing to start?

**Pull request merged**